### PR TITLE
Improved visuals of main menu and some UX features

### PR DESCRIPTION
**Pull request by w050asg**
Altered the look of the main menu by adding a title and using different colors to improve the look. Got strings to display to allow the user to determine what each button does. UX improvements like opening in fullscreen and exiting on escape to make it easier for the user.

### CODE REVIEW

**Comment by NicholaiThe3rd** [comment]
Looks good so far, still attempting to find a way to implement these as buttons but I appreciate how clean it looks now. Possibly make the last section as an "Exit" button instead since we currently don't have any plans to add an options menu at this point. 

If some more comments to help the documentation would be possible I would appreciate it.

File: src/main.c [hunk 1]
```diff
@@ -2624,33 +2629,61 @@ void render(void){
...
     glTranslatef(0.0, 0.1, 0.0);
-    display();
+    renderButtonBackground();
+    glPopMatrix();
+
+    glPushMatrix();
+    glLoadIdentity();
+    glColor3f(1.0f,1.0f,1.0f);
+    glRasterPos2f(0.45, 0.76);
+    glutBitmapString(GLUT_BITMAP_TIMES_ROMAN_24, "Play Game");
```
Comment (jrichards1999) [inline src/main.c:2648]: This looks like a pretty convenient way to display text, wish I had used it for my project!

Comment (jrichards1999) [inline src/main.c:2648]: This explains why the menu pops up at the start of main.  Sorry for missing that earlier!

File: src/main.c [hunk 2]
```diff
@@ -2624,33 +2629,61 @@ void render(void){
...
 }
 
-
 void game_loop() {
     // startCraft(argc, **argv);
 glutInit(&argc, argv);
 glutInitDisplayMode (GLUT_SINGLE | GLUT_RGB);
-glutInitWindowSize (800, 600);
-glutInitWindowPosition (100, 100);
+glutInitWindowSize (glutGet(GLUT_SCREEN_WIDTH), glutGet(GLUT_SCREEN_HEIGHT));
```
Comment (jrichards1999) [inline src/main.c:2682]: Good use of descriptive variables here to make the code more readable.

**Review by jrichards1999 — COMMENTED** [review]
Great work, Austin.  The changes made improve code readability, but could still use some extra comments to help explain what is happening in the renderButton function.

**Review by NicholaiThe3rd — COMMENTED** [review]
Looks good so far, still attempting to find a way to implement these as buttons but I appreciate how clean it looks now. Possibly make the last section as an "Exit" button instead since we currently don't have any plans to add an options menu at this point.

If some more comments to help the documentation would be possible I would appreciate it.

**Pull request merged**